README.md

```diff
@@ -1,2 +1,29 @@
-# hw01
-Repository for first CFSS homework assignment
+# HW 01 #
+<hr />
+## Short biography (*of sorts*) ##
+<hr />
+ * **Name** 
+          <br /> &nbsp; &nbsp;&nbsp;&nbsp; Esha Banerjee 
+ * **Motto**
+            <br /> &nbsp; &nbsp;&nbsp;&nbsp; "I love sleep. My life has the tendency to fall apart when I’m awake, you know?" <small> -Ernest Hemingway </small>
+ * **Hobbies** 
+     * Giving opinions (..everyone has one :wink:)
+     * Trekking cause mountains are :heart:
+     * Cooking *(Fattening others improves self-esteem)* 
+     * Collecting trivia
+     <br /> &nbsp;&nbsp;*UChicago fun fact:* The university logo in use today was introduced in Fall 2012. Here's the old logo:
```

Comment:
I think the indent here is random, but again, might just be a preference. I think its cool you put a picture in :)